authenticate.go

```diff
@@ -226,6 +187,127 @@ func (c *OktaClient) cancelCurrentFactor(transaction api.AuthenticationTransacti
 	return c.handleAuthUserFlow(newTransaction, false)
 }
 
+func (c *OktaClient) handleFactorTypeU2F(transaction api.AuthenticationTransaction) (string, error) {
```

Comment:
This was totally my bad cause I left that comment asking for this (and nothing actionable here), but I think in the future I would have broken up this change of pulling each of the factors out as its own PR for easy viewing.